src/utils/object.js

```diff
@@ -86,4 +86,10 @@ export const omit = (obj, props) =>
     .filter(key => props.indexOf(key) === -1)
     .reduce((result, key) => ({ ...result, [key]: obj[key] }), {})
 
+// @link https://gist.github.com/bisubus/2da8af7e801ffd813fab7ac221aa7afc
+export const pick = (obj, props) =>
+  Object.keys(obj)
```

Comment:
Should use the `keys` object helper method

Also this is similar to the pluck-props utility